daemon/api_model.go

```diff
@@ -69,3 +90,73 @@ func postModel(c *Command, r *http.Request, _ *auth.UserState) Response {
 	return AsyncResponse(nil, &Meta{Change: chg.ID()})
 
 }
+
+// getModel gets the current model assertion using the DeviceManager
```

Comment:
these needs some unit tests

Reply:
I added unit tests for getting the model without a model assertion and with a model assertion, and also that the endpoint returns the new model assertion after a remodel, however the last test doesn't seem to be passing, could you take a look at it and see what I should be doing in the mock to make the new model be taken into account in the device manager?

Reply:
I also couldn't find any examples of mocking serial assertions, any pointers for where I could look to mock a serial assertion for testing as well?

Reply:
looks for

```
serial, err := s.brands.Signing("my-brand").Sign(asserts.SerialType, map[string]interface{}{...
```

in overlord/managers_test.go for example

Reply:
Alright, I dropped my test around /v2/model showing the new assertion, that ended up being a lot more complicated than I think is really necessary.
I also added new unit tests around getting a serial assertion so I think the test coverage should be adequate, let me know if you disagree.